input/pagecontent/downloads.md

```diff
@@ -9,31 +9,31 @@ It is not recommended to view locally without hosting, but you can extract the f
 
 ### Package file
 
-<!-- The following package file includes an NPM package file used by many of the FHIR tools.  It contains all the value sets, profiles, extensions, list of pages and urls in the IG, etc defined as part of this version of the Implementation Guides. This file should be the first choice whenever generating any implementation artefacts since it contains all of the rules about what makes the profiles valid. Implementers will still need to be familiar with the content of the specification and profiles that apply in order to make a conformant implementation:
+The following package file includes an NPM package file used by many of the FHIR tools.  It contains all the value sets, profiles, extensions, list of pages and urls in the IG, etc defined as part of this version of the Implementation Guides. This file should be the first choice whenever generating any implementation artefacts since it contains all of the rules about what makes the profiles valid. Implementers will still need to be familiar with the content of the specification and profiles that apply in order to make a conformant implementation:
 
 - [R4 Package](package.tgz){::download="true"}
 - [R4B Package](package.r4b.tgz){::download="true"}
 
-See the overview on [validating FHIR profiles and resources](http://hl7.org/fhir/R4/validation.html) for more information about validating profiles and how to use these artefacts. -->
+See the overview on [validating FHIR profiles and resources](http://hl7.org/fhir/R4/validation.html) for more information about validating profiles and how to use these artefacts.
 
 ### Examples 
 
-<!-- All examples included in this implementation guide are available for download:
+All examples included in this implementation guide are available for download:
 
 - [XML](examples.xml.zip)
 - [JSON](examples.json.zip)
 - [TTL](examples.ttl.zip) -->
```

Comment:
two instances of comment end after TTL --> need to be removed, these are currently being rendered in the html

Reply:
Removed.